SQLTools.sublime-settings

```diff
@@ -68,153 +68,232 @@
+                },
+                "columns": {
+                    "query": "select '|' || quote_ident(table_name) || '.' || quote_ident(column_name) || '|' from information_schema.columns where table_schema not in ('pg_catalog', 'information_schema') order by table_name, ordinal_position;",
+                    "options": ["--tuples-only", "--no-psqlrc"]
+                },
+                "functions": {
+                    "query": "select '|' || quote_ident(n.nspname)||'.'||quote_ident(f.proname) || '(' || pg_get_function_identity_arguments(f.oid) || ')' || '|' as funname from pg_catalog.pg_proc as f inner join pg_catalog.pg_namespace as n on n.oid = f.pronamespace where f.proisagg = false and n.nspname not in ('pg_catalog', 'information_schema');",
+                    "options": ["--tuples-only", "--no-psqlrc"]
                 }
             }
         },
         "oracle": {
             "options": ["-S"],
             "before": [
-                "SET AUTO           OFF",
-                "SET COLSEP         '|'",
-                "SET FEED           ON",
-                "SET FEEDBACK       ON",
-                "SET HEADING        ON",
-                "SET LINESIZE       32767",
-                "SET LONG           100",
-                "SET NULL           @",
-                "SET PAGESIZE       0 EMBEDDED ON",
-                "SET SERVEROUTPUT   ON",
-                "SET SQLBLANKLINES  ON",
-                "SET SQLPROMPT      ''",
-                "SET TAB            OFF",
-                "SET TI             ON",
-                "SET TIMI           OFF",
-                "SET TRIMSPOOL      OFF",
-                "SET UND            '-'",
-                "SET VERIFY         OFF ",
-                "SET WRAP           OFF"
+                "SET LINESIZE   32767",
+                "SET WRAP       OFF",
+                "SET PAGESIZE   0",
+                "SET EMBEDDED   ON",
+                "SET TRIMOUT    ON",
+                "SET TRIMSPOOL  ON",
+                "SET TAB        OFF",
+                "SET SERVEROUT  ON",
+                "SET NULL       '@'",
+                "SET COLSEP     '|'",
+                "SET SQLBLANKLINES ON"
             ],
+            "after": [],
+            "env_optional": {
+                "NLS_LANG": "{nls_lang}"
+            },
             "args": "{username}/{password}@\"(DESCRIPTION=(ADDRESS_LIST=(ADDRESS=(PROTOCOL=TCP)(HOST={host})(PORT={port})))(CONNECT_DATA=(SERVICE_NAME={service})))\"",
             "queries": {
-                "desc" : {
-                    "query": "select concat(concat(concat(concat('|', owner), '.'), table_name), '|') as tbls from all_tables UNION ALL select concat(concat(concat(concat('|', owner), '.'), view_name), '|') as tbls from all_views;",
-                    "options": ["-S"]
+                "execute": {
+                    "options": [],
+                    "before": [
+                        // "SET TIMING     ON",
```

Comment:
I guess we can remove this comment

Reply:
I left this option in the comments intentionally, as I thought this might be a popular option which some people might be looking for.
But if you say so, I can easily remove it.